### ISSUE TITLE
Training on small data set

### ISSUE THREAD

**Issue by martin-radfar** [kaituoxu/SpeechTransformer#26]
Hi Kaituo,
I'm doing a very naive test. I selected one speaker from aishell(spkr = 2,#snt = 364, #wrd = 4,989) and used this speaker both for training, dev, and test. I expected to get some decent results  becuase I'm overfitting the model. However here are the results I got (#epoch = 50):

 SPKR           |        # Snt                # Wrd         |        Corr                   Sub                  Del                   Ins                   Err                S.Err         |
|        Sum/Avg        |         364                  4989         |         0.2                  95.9                  3.9                   3.9                 103.7                100.0 

I understand  an NN  needs much much more data to work. but I wonder why the results are poor for this toy example when we use the same data for train, dev and test. shouldn't at least we get better results.

**Comment by kaituoxu** [comment]
Try to close some regularization methods including label smoothing, dropout...
Then adjust `k` related with learning rate.

**Comment by martin-radfar** [comment]
I did based on the previous post but still error rate is 100 . I used 150
epochs. What k value would you choose ? May be irrelevant but I use cup .

**Comment by kaituoxu** [comment]
Try different `k` or `warmup_steps`, they will effect learning rate, this is very very important.
